.github/workflows/continous-integration.yml

```diff
@@ -0,0 +1,35 @@
+name: Continuous Integration
+
+on:
+  push:
+    branches:
+    - main
+    tags:
+    - '*'
+  pull_request:
+
+jobs:
```

Comment:
if you want we can also add publishing to avoid doing this manually.

Reply:
Not sure what you mean, but my knowledge of gh workflows is pretty lacking

Reply:
Sorry, I was also very vague 🤦🏻 What I meant is that we can add the publishing of your package to pypi to the GitHub actions. Means that you can just do something like `git tag 1.2.3` and `git push --tags` and it will automatically publish your package to pypi

Reply:
Ah I see. I actually prefer to do it manually right now so that I have control over publishing. Also I haven't really used git tags so it might take me a minute to get used to them. But thanks I didn't know this was a workflow that ppl used! I'll have to look into it